python/ray/_private/structured_logging/logging_config.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from ray._private.structured_logging.constants import LOG_MODE_DICT
+from ray.util.annotations import PublicAPI
+
+
+@PublicAPI(stability="alpha")
+class LoggingConfig:
+    def __init__(self, encoding: Optional[str] = "TEXT", log_level: str = "INFO"):
+        """
```

Comment:
Let's follow https://docs.ray.io/en/latest/ray-contribute/getting-involved.html#code-style, the class docstring should be before `__init__`

Reply:
Updated https://github.com/ray-project/ray/pull/45344/commits/c8621a0476b43870f936da4908a64f3e8cb052e4